src/fu-util.c

```diff
@@ -2011,6 +2011,7 @@ fu_util_switch_branch (FuUtilPrivate *priv, gchar **values, GError **error)
 	g_signal_connect (priv->client, "device-changed",
 			  G_CALLBACK (fu_util_update_device_changed_cb), priv);
 	priv->flags |= FWUPD_INSTALL_FLAG_ALLOW_REINSTALL;
+	priv->flags |= FWUPD_INSTALL_FLAG_ALLOW_BRANCH_SWITCH;
```

Comment:
I guess it's worth asking - why not also have this same argument on `fwupdtool`?  Installing a CAB could encounter the same situation.

Reply:
You're completely correct. Fixed.